### DISCUSSION TITLE
We want more educational videos or we want someone who is a professional in serenity explaining a project to us because the Internet is poor in serenity content

### DISCUSSION THREAD

**Discussion by ra-bedo11** [serenity-is/Serenity#6345] [Help]
We want more educational videos or we want someone who is a professional in serenity explaining a project to us because the Internet is poor in serenity content

**Comment by wirble** [comment]
The developers have a pricing plan and can answer more detail questions than what free can provide.  https://serenity.is/#Pricing
Have you reached out to them to ask for your *Want*?
